src/datasources/balances-api/coingecko-api.interface.ts

```diff
@@ -0,0 +1,18 @@
+import { CoingeckoAssetPrice } from './entities/coingecko-asset-price.entity';
+
+export const ICoingeckoApi = Symbol('ICoingeckoApi');
+
+export interface ICoingeckoApi {
```

Comment:
Nit: I think we should genericise this interface name.

Reply:
I agree. Giving it a second thought, I've restored the original name (`IPricesApi`) 

Changed in [a62b130](https://github.com/safe-global/safe-client-gateway/pull/1175/commits/a62b1305da4626e6b4362831001b84289acfed44)